CDS/js/test/TestMediaAtomLib.js

```diff
@@ -122,7 +122,7 @@ describe('mediaAtomLib', () => {
                         'X-Gu-Tools-Service-Name': 'content_delivery_system'
                     }})
                 .put(URI, {
-                    uri: 'youtube_url'
+                    uri: 'https://www.youtube.com/watch?v=youtube_id'
```

Comment:
Sorry, I'm not sure what you mean. This is not actually posting anything, but it tests what request was made. the id of the asset in the test is just youtube_id, so this is then the uri that should get posted?